source/declarations.tex

```diff
@@ -8512,6 +8512,14 @@
 behavior is \impldef{behavior of non-standard attributes}.
 Any \grammarterm{attribute-token} that is not recognized by the implementation
 is ignored.
+An \grammarterm{attribute-token} is reserved for future standardization if
+\begin{itemize}
+\item it is not an \grammarterm{attribute-scoped-token} and
+not specified in this document, or
+\item it is an \grammarterm{attribute-scoped-token} and
+its \grammarterm{attribute-namespace} is
+either \tcode{std} or \tcode{std} followed by one or more digits.
+\end{itemize}
```

Comment:
#3127 suggests that a blank line should precede and follow each of these \items (note that I've interpreted that to include the \begin{itemize}/\end{itemize} as part of the first/last \item)